lib/segment/src/index/field_index/map_index/mod.rs

```diff
@@ -1242,6 +1340,18 @@ mod tests {
                 }
                 builder.finalize().unwrap();
             }
+            IndexType::MutableGridstore => {
+                let mut builder = MapIndex::<N>::builder_gridstore(path.to_path_buf());
+                builder.init().unwrap();
+                for (idx, values) in data.iter().enumerate() {
+                    let values: Vec<Value> = values.iter().map(&into_value).collect();
+                    let values: Vec<_> = values.iter().collect();
```

Comment:
Why are we collecting here twice?

Reply:
We need a `&[&Value]` there, but we need to convert it into a owned `Value` first. So that's why there's a two step approach.

Note that this was here before, and I only copy-pasted it :smiley: